### ISSUE TITLE
Feedback: Content is "hidden" in settings folder

### ISSUE THREAD

**Issue by DennisL68** [estruyf/vscode-front-matter#468]
Default content folder for markdown files for the FrontMatter project is placed in the `.frontmatter` folder which I first perceived as a settings folder.  
The frontmatter.json settings file is placed in the project root folder (which I'm quite happy with).

But shouldn't the root folder for the project also be the default content folder?
What is the purpose of hiding the content folder?

**Comment by DennisL68** [comment]
Ok, I see. The folder `content` actually contains content **type** origin documents?

**Comment by estruyf** [comment]
Not necessarily; you can put these there. It is how you wish to link these.

**Comment by estruyf** [comment]
Yes, the `frontmatter.json` file is supposed to be outside of the `.frontmatter` folder.
